announcements/release-7.0.html

```diff
@@ -109,11 +109,13 @@ <h1>Astropy v7.0 Released!</h1>
           <li>New Lorentz2D model</li>
           <li>Faster guessing of formats in astropy.io.ascii</li>
           <li>Support VOTable version 1.5</li>
+          <li>New `parquet.votable` format is available to read/write a Table from parquet files with VOTable metadata included</li>
```

Comment:
I suppose we need to upper case it. I'm sorry about it.
```suggestion
          <li>New `parquet.votable` format is available to read/write a Table from Parquet files with VOTable metadata included</li>
```

Reply:
no problem, I leave open for a bit so more eyes can check this kind of things ;)